src/tests/end-to-end/common/page-controllers/page.ts

```diff
@@ -125,6 +131,10 @@ export class Page {
         return this.waitForSelector(`#${id}`);
     }
 
+    public async waitForDuration(duration: number): Promise<void> {
```

Comment:
These tests are no longer using any of these new methods defined on `Page`; waitForNavigation isn't contentious, but `waitForDuration` and `frames` are probably methods we'd prefer tests avoid the temptation to call, so I'd be inclined to leave them out if they aren't necessary, and `takeScreenshot` seems redundant

Reply:
removed the extra ones.